src/features/auth/components/LoginForm.tsx

```diff
@@ -47,6 +63,28 @@ export const LoginForm = ({
     openForgotPasswordForm()
     onClose()
   }
+  const handleSubmit = (values: LoginUser) => {
+    setErrorMessage(DEFAULT_ERROR_MESSAGE)
+    mutate(
+      { userData: values },
+      {
+        onSuccess: () => {
+          onClose()
+          reloadUser()
+          router.push(routes.myProjects())
+        },
+        onError: (error) => {
+          if (
+            error instanceof AxiosError &&
+            error?.response?.status?.toString().startsWith('4')
```

Comment:
The endpoint you're calling can only return two 4xx codes: 400 (invalid request) or 401 (login failed). Handle only these two cases. If somebody (maybe even me 😅) messes up on backend and returns some other 4xx error code, then that would be misleading to display this message.